bindings/rust/bench/README.md

```diff
@@ -14,20 +14,20 @@ For example, to get started with benching s2n-tls with AWS-LC:
 
 ```
 ../generate.sh
-certs/generate_certs.sh
-./install-aws-lc.sh
+scripts/generate_certs.sh
+scripts/install-aws-lc.sh
 cargo bench --config aws-lc-config/s2n.toml
 ```
 
 ## Running benchmarks
 
 The benchmarks can be run with the `cargo bench` command. Criterion will auto-generate an HTML report in `target/criterion/`. 
 
-To run memory benchmarks, run `memory/bench-memory.sh`. A graph of memory usage will be generated in `memory/memory.svg`.
+To run memory benchmarks, run `bench-memory.sh`. A graph of memory usage will be generated in `memory/memory.svg`.
 
 ## Historical benchmarks
 
-To do historical benchmarks, run `historical-perf/bench-past.sh`. This will checkout old versions of s2n-tls back to v1.3.16 in `target/` and run benchmarks on those with the `historical-perf` feature, disabling Rustls and OpenSSL benches.
+To do historical benchmarks, run `bench-past.sh`. This will checkout old versions of s2n-tls back to v1.3.16 in `target/` and run benchmarks on those with the `historical-perf` feature, disabling Rustls and OpenSSL benches.
```

Comment:
Do you wan to mention that the bench-past is in `scripts/bench-past`. As a quick start guide maybe at the top you can have a copy/paste version of this:


### Quickstart
memory: `./scripts/bench-memory.sh`
historical: ...

Reply:
I wasn't sure whether or not I should've included `scripts/` before each shell script name, but since it seems like it'd make it clearer I'll add that. Also just moved the example code that was in the readme up to be more obvious in the quickstart.